_inc/client/writing/custom-content-types.jsx

```diff
@@ -19,55 +23,57 @@ import SettingsGroup from 'components/settings-group';
 export class CustomContentTypes extends React.Component {
 	state = {
 		testimonial: this.props.getOptionValue( 'jetpack_testimonial', 'custom-content-types' ),
-		portfolio: this.props.getOptionValue( 'jetpack_portfolio', 'custom-content-types' )
+		portfolio: this.props.getOptionValue( 'jetpack_portfolio', 'custom-content-types' ),
 	};
 
 	updateCPTs = type => {
-		const deactivate = 'testimonial' === type
-			? ! ( ( ! this.state.testimonial ) || this.state.portfolio )
-			: ! ( ( ! this.state.portfolio ) || this.state.testimonial );
+		const deactivate =
+			'testimonial' === type
+				? ! ( ! this.state.testimonial || this.state.portfolio )
+				: ! ( ! this.state.portfolio || this.state.testimonial );
 
 		this.props.updateFormStateModuleOption( 'custom-content-types', 'jetpack_' + type, deactivate );
 
 		this.setState( {
-			[ type ]: ! this.state[ type ]
+			[ type ]: ! this.state[ type ],
 		} );
 	};
 
 	linkIfActiveCPT = type => {
-		return this.props.getSettingCurrentValue( 'jetpack_' + type, 'custom-content-types' )
-			? <a href={ this.props.siteAdminUrl + 'edit.php?post_type=jetpack-' + type } />
-			: <span />;
+		return this.props.getSettingCurrentValue( 'jetpack_' + type, 'custom-content-types' ) ? (
+			<a href={ this.props.siteAdminUrl + 'edit.php?post_type=jetpack-' + type } />
```

Comment:
Could we use template literals here to make it easier to read?

Reply:
Addressed in 00c07cd471ad1d629f1edf72f3cfdd04b25f318a